docs/sagemaker.md

```diff
@@ -29,7 +29,7 @@ Here's a list of frameworks and versions which support this experience.
 | [TensorFlow](tensorflow.md) | 1.15 |
 | [MXNet](mxnet.md) | 1.6 |
 | [PyTorch](pytorch.md) | 1.3 |
-| [XGBoost](xgboost.md) | |
+| [XGBoost](xgboost.md) | >=0.90-2 [As Built-in algorithm](xgboost.md#use-xgboost-as-a-built-in-algorithm)|
```

Comment:
Will this be >= or just == ?

Reply:
from xgboost doc `version `0.90-2` or later`